Add reference to sample repo (#154)
Azure/azure-iot-middleware-freertos: README.md

## Repo Structure

> This repo contains the Azure IoT Middleware for FreeRTOS libraries and carries dependencies only to coreMQTT. For working samples please visit the [samples repo](https://github.com/Azure-Samples/iot-middleware-freertos-samples). There we have samples for several devices including:

Reviewer: ```suggestion
This repo contains the Azure IoT Middleware for FreeRTOS libraries and has dependencies only on coreMQTT. For working samples please visit the [samples repo](https://github.com/Azure-Samples/iot-middleware-freertos-samples). There we have samples for several devices including:
```
